cameo/modeling-requirements.in

```diff
@@ -0,0 +1,16 @@
+# Include the base requirements, but NOT the pre-compiled constraints. If there
+# is a reason to do re-use those, the following incompatibility (and possibly
+# more) in the resolved dependencies would have to be adressed:
+#   pytest==5.2.2 (from -r /opt/base-requirements.txt (line 297))
+#   pytest<5,>=4.0.1 (from escher==1.7.3->cameo==0.11.15->-r modeling-requirements.in (line 7))
```

Comment:
cameo we could easily address but escher I have no clue who maintains that now.

Reply:
Isn't Zak maintainer of the Python version too? Maybe escher could be an optional dependency in cameo?

Regardless, should we move forwards with my idea of not using the explicit constraints until that's potentially resolved?